### PR TITLE
[sos] Fix typos in sos help

### PR DESCRIPTION
**Pull request by dwolstroRH**
Related: RHELPLAN-170117

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [X] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?
- [X] Are all passwords or private data gathered by this PR [obfuscated](https://github.com/sosreport/sos/wiki/How-to-Write-a-Plugin#how-to-prevent-collecting-passwords)?

### CODE REVIEW

**Review by jcastill — APPROVED** [review]
LGTM

File: sos/report/__init__.py [hunk 1]
```diff
@@ -421,9 +421,9 @@ def display_help(cls, section):
             'Plugins may specify different collections on different distribu'
             'tions, and some plugins may only be for specific distributions. '
             'Distributions are represented within SoS by \'policies\' and may '
-            'influence how other SoS commands or options function. For example'
-            'policies can alter where the --upload option defaults to or '
-            'functions.'
+            'influence how other SoS commands or options function. For '
+            'example policies can alter where the --upload option defaults '
+            'to or functions.'
```
Comment (arif-ali) [inline sos/report/__init__.py:426]: This doesn't look like a typo to me, it's still the same tring 🤷🏽

Comment (jcastill) [reply sos/report/__init__.py:426]: It should be the space between example and policies. If you run 'sos help report' you get:

```
For examplepolicies can alter where the --upload option defaults
to or functions.
```

That this change should fix.

Comment (arif-ali) [reply sos/report/__init__.py:426]: Yup, I see it now

**Review by pmoravec — APPROVED** [review]

**Review by TurboTurtle — APPROVED** [review]

**Pull request merged**